CONTRIBUTING.md

```diff
@@ -1,9 +1,10 @@
 # Instructions for Contributors
 
-As mentioned in the [README](https://github.com/haskell/containers/blob/master/README.md), for reporting bugs (and maybe even the respective fix), please use the [GitHub issue tracker](https://github.com/haskell/containers/issues).
+For reporting bugs (and maybe even the respective fix), please use the [GitHub issue tracker](https://github.com/haskell/containers/issues).
```

Comment:
Reword this. Perhaps something like this:

> To report bugs, please use the [GitHub issue tracker](https://github.com/haskell/containers/issues). We also appreciate [pull requests](https://github.com/haskell/containers/pulls) on GitHub.

Reply:
Done.